post_installation_configuration/multi-architecture-configuration.adoc

```diff
@@ -1,21 +1,21 @@
 :_content-type: ASSEMBLY
 :context: multi-architecture-configuration
 [id="post-install-multi-architecture-configuration"]
-= Configuring a multi-architecture cluster
+= Configuring multi-architecture compute machines on an {product-title} cluster 
 include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-A multi-architecture cluster is a cluster that supports worker machines with different architectures. You can deploy a multi-architecture cluster by creating an Azure installer-provisioned cluster using the multi-architecture installer binary. For Azure installation, see xref:../installing/installing_azure/installing-azure-customizations.adoc[Installing a cluster on Azure with customizations].
+A {product-title} cluster with multi-architecture compute machines is a cluster that supports compute machines with different architectures. You can deploy a cluster with multi-architecture compute machines by creating an Azure installer-provisioned cluster using the multi-architecture installer binary. For Azure installation, see xref:../installing/installing_azure/installing-azure-customizations.adoc[Installing a cluster on Azure with customizations].
 
 [WARNING]
 ====
-The multi-architecture clusters Technology Preview feature has limited usability with installing, upgrading, and running payloads. 
+The multi-architecture compute machines Technology Preview feature has limited usability with installing, upgrading, and running payloads. 
 ====
 
-The following procedures explain how to generate an `arm64` boot image and create an Azure compute machine set with the `arm64` boot image. This will add `arm64` worker nodes to your multi-architecture cluster and deploy the desired amount of ARM64 virtual machines (VM). This section also shows how to upgrade your existing cluster to a multi-architecture cluster. Multi-architecture clusters are only available on Azure installer-provisioned infrastructures with `x86_64` control planes. 
+The following procedures explain how to generate an `arm64` boot image and create an Azure compute machine set with the `arm64` boot image. This adds `arm64` compute nodes to your cluster and deploys the desired amount of `arm64` virtual machines (VM). This section also shows how to upgrade your existing cluster to a cluster with multi-architecture compute machines. Clusters with multi-architecture compute machines are only available on Azure installer-provisioned infrastructures with `x86_64` control plane machines. 
 
```

Comment:
should be: "This section also shows how to upgrade your existing cluster to a cluster which supports multi-architecture compute machines"